components/Switch.jsx

```diff
@@ -106,3 +117,70 @@ Switch.propTypes = {
   onChange: PropTypes.func,
   styles: PropTypes.object
 }
+
+const renderSwitch = (name, checked, children, classNames, onClick, onMouseDown, onMouseUp, remainingProps) => (
+  <div
+    className={classNames}
+    onClick={onClick}
+    onMouseDown={onMouseDown}
+    onMouseUp={onMouseUp}
+    {...remainingProps}>
+    {children}
+    {name &&
+      <input name={name} type='hidden' value={checked} />}
+  </div>
+)
+
+const renderDynamicallyStyledSwitch = (name, checked, children, classNames, childClassNames, onClick, onMouseDown, onMouseUp, { backgroundColor, bulletColor }, remainingProps) => (
+  <div
+    className={classNames}
+    onClick={onClick}
+    onMouseDown={onMouseDown}
+    onMouseUp={onMouseUp}
+    {...remainingProps}>
+    <div
+      className={childClassNames.bullet}
+      style={{
+        backgroundColor: backgroundColor,
+        borderColor: backgroundColor
+      }}></div>
```

Comment:
No, not at all. The CSS components don't currently support that, but we should most definitely implement it as a regular one.